modules/ROOT/pages/cloudhub-setup.adoc

```diff
@@ -56,12 +56,7 @@ For more information about Anypoint Monitoring, see xref:monitoring::enable-apps
 
 == Verify Supported Mule Versions for Policies
 
-The APIs and policies that Anypoint Visualizer displays in the Policy Visualization come from an update to the agent in Mule runtime engine (Mule). As a result, policies are shown in Anypoint Visualizer only for applications running on these Mule versions:
-
-* Mule 3.9.3+ patch releases available on December 12, 2019 or later.
-* Support for Mule 4.2.x will be available in the next patch update.
-
-// add! dates! above! dependency
+The APIs and policies that Anypoint Visualizer displays in the Policy visualization come from an update to the agent in Mule runtime engine (Mule). As a result, policies are shown in Anypoint Visualizer only for applications running on any of the patches of Anypoint Visualizer supported Mule versions available on December 12, 2019 or later.
```

Comment:
Should visualization be init cap?
I had to read the 2nd sentence twice. Maybe "on Mule version patches supported by Anypoint Visualizer that were released on xxx and later"?

Reply:
No, that doesn't work either, I see.
I would say add a hyphen after Visualizer (Visualizer-supported), but I think we can't do that.
Anypoint Visualizer shows policies only for apps running on supported Mule versions released on xxxx.

Reply:
```suggestion
The APIs and policies that Anypoint Visualizer displays in the Policy visualization come from an update to the agent in Mule runtime engine (Mule). As a result, Anypoint Visualizer shows policies only for apps running on supported Mule versions released on December 12, 2019 or later.
```